vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -528,7 +528,10 @@ - (void)fetchAvailableGrantsForLanguage:(NSString *)language paymentId:(NSString
   //  - OnGrant (GRANT_NOT_FOUND, LEDGER_ERROR, LEDGER_OK)
   // Calls `OnGrant` for each grant found (...)
   ledger->FetchGrants(std::string(language.UTF8String),
-                      std::string(paymentId.UTF8String));
+                      std::string(paymentId.UTF8String),
+                      ^(ledger::Result result){
+    // handle this?
```

Comment:
For this method we'll want to add a callback to the actual containing method i.e.

`fetchAvailableGrantsForLanguage:paymentId:completion:` as seen in other methods with callbacks. Changes will have to be made to the declaration in `BATBraveLedger.h` as well. Let me know if you need assistance with that